Automattic/wp-calypso
# Goals and DP first: Move DIFM intro before account

Closes https://github.com/Automattic/wp-calypso/issues/97466.

## Proposed Changes

Show the DIFM intro step before redirecting the user to account creation.

## Why are these changes being made?

So there's more context about DIFM before showing the user the sign up step.

## Testing Instructions

1. Enter `/setup/onboarding`
2. Click the "Let us build a custom site for you" link

- Verify you see the DIFM intro screen and click "Get started"
- Verify you get redirected to the login screen
- Verify you enter the DIFM flow after signing in
